specification/appconfiguration/resource-manager/Microsoft.AppConfiguration/preview/2024-09-01-preview/appconfiguration.json

```diff
@@ -0,0 +1,3168 @@
+      "post": {
+        "tags": [
+          "ConfigurationStores"
+        ],
+        "description": "Generates the SAS token for scoped, read-only access of the specified configuration store.",
+        "operationId": "ConfigurationStores_GenerateSasToken",
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ConfigStoreNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "name": "generateSasTokenParameters",
+            "in": "body",
+            "description": "The object containing information for the SAS token generation request.",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/GenerateSasTokenParameters"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The request was successful; the request was well-formed and received properly.",
+            "schema": {
+              "$ref": "#/definitions/SasTokenGenerateResult"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "ConfigurationStores_ListKeys": {
+            "$ref": "./examples/ConfigurationStoresGenerateSasToken.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.AppConfiguration/configurationStores/{configStoreName}/resetSas": {
+      "post": {
+        "tags": [
+          "ConfigurationStores"
+        ],
+        "description": "Reset the SAS token of a given type.",
+        "operationId": "ConfigurationStores_ResetSas",
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ConfigStoreNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "name": "resetSasParameters",
+            "in": "body",
+            "description": "The object containing information for the SAS token reset request.",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/ResetSasParameters"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The request was successful; the request was well-formed and received properly."
```

Comment:
Is it odd that this API returns no response body?

Reply:
I think it's fine for a "POST" operation. For example, purge post operation also returns no response body.